packages/react/src/components/ComposedModal/ComposedModal.tsx

```diff
@@ -421,7 +421,7 @@ const ComposedModal = React.forwardRef<HTMLDivElement, ComposedModalProps>(
         ref={ref}
         aria-hidden={!open}
         onBlur={!focusTrapWithoutSentinels ? handleBlur : () => {}}
-        onMouseDown={handleMousedown}
+        onClick={handleClick}
```

Comment:
Before this change a consumer could pass in `onClick`, but with this change they'll no longer be able to do that. The reverse is also true for `onMouseDown`, but that's going to be less of a concern. I doubt anyone has placed `onMouseDown` on a modal, because it would never have been called before this change.

To fix we can either conditionally call rest.onClick from within `handleClick`, or use the composeEventHandlers helper we have. I think we could use optional chaining to avoid errors for when they're not supplying their own `onClick`? 🤔 

```js
onClick={composeEventHandlers([rest.?onClick, handleClick])}
```

Reply:
great point! will fix 👍